web/server/routers/account.ts

```diff
@@ -218,8 +217,25 @@ export const accountRouter = router({
     .input(searchInput)
     .query(async ({ input, ctx }) => {
       const session = await Account.startSession();
```

Comment:
Nit: probably unnecessary to use a transaction here since we are only making 1 database mutation.